src/cli/main.pl

```diff
@@ -980,50 +986,53 @@
 
     create_authorization(Opts,Authorization),
 
+    format(current_output, "Cloning: origin~n", []),
```

Comment:
the message formatting suggests this could say something else than 'origin' but that's not the case. Maybe this should give some more detail like 'cloning from <remote> into new database <db>'?

Reply:
The remote name `origin` is hardcoded for the `clone` operation but not for other operations. I added it here, because you otherwise do not know which remote name is being used. (I think we eventually should add a flag for the remote name to be consistent with other commands. I've created #1167 for that.).

It does say `Database created` after the operation is completed below. I used `Database created` to be consistent with the message of `db create`.

The URL cloned from is in the command argument, so I elided that to avoid redundancy. But if you think we should put the URL here, I'm fine with that.

Would it be better for you if the message was `Cloning from remote 'origin'`?

Reply:
> Would it be better for you if the message was `Cloning from remote 'origin'`?
That'd be better. Even better imo would be to include the URI that is being cloned, but that's nitpicking.

Reply:
> Even better imo would be to include the URI that is being cloned, but that's nitpicking.

Do you mean the remote URI that is passed to the `clone` command? As I said, that's just a bit redundant, because it's already there, but I'm fine with it. If we do that with `clone`, then I would also change the other commands to reprint some of the arguments for consistency.

Or do you mean the actual API URI used internally? If so, that might involve changing the `clone` predicate in `db_clone.pl`